src/selectors/pipeline.js

```diff
@@ -49,3 +51,24 @@ export const getPipelineTagIDs = createSelector(
     return Object.keys(visibleTags);
   }
 );
+
+/**
+ * Get IDs of modular pipelines used in the current pipeline
+ */
+export const getPipelineModularPipelineIDs = createSelector(
+  [getPipelineNodeIDs, getNodeModularPipelines, getModularPipelineIDs],
+  (nodeIDs, nodeModularPipelines, modularPipelineIDs) => {
+    const visibleModularPipelines = {};
+    // check if pipeline contains defined modular pipelines
+    if (modularPipelineIDs.length > 0) {
+      nodeIDs.forEach((nodeID) => {
+        nodeModularPipelines[nodeID].forEach((modularPipelineID) => {
+          if (!visibleModularPipelines[modularPipelineID]) {
```

Comment:
This check `if (!visibleModularPipelines[modularPipelineID])` seems like it can be removed to simplify a bit since it only sets `true`?